### PR TITLE
Update: AP_Periph Hwdef Carbonix l496

### PR DESCRIPTION
**Pull request by Lokesh-Carbonix**
CarbonixL496 hwdef file update to integrate below settings
GPS
ADSB
JTAG
LED
Clock Freq

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef-bl.dat [hunk 1]
```diff
@@ -61,8 +62,8 @@ PB8 CAN1_RX CAN1
 PB9 CAN1_TX CAN1
 
 # debugger support, disabled as PA13 used for LED
```
Comment (amilcarlucas) [inline libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef-bl.dat:64]: You should also remove this line

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef.dat [hunk 2]
```diff
@@ -79,8 +80,8 @@ I2C_ORDER I2C4
 define HAL_PERIPH_LISTEN_FOR_SERIAL_UART_REBOOT_CMD_PORT 0
 
 # debugger support (disabled as conflicts with LED)
```
Comment (amilcarlucas) [inline libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef.dat:82]:
```suggestion
# debugger support
```

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef.dat [hunk 3]
```diff
@@ -18,8 +18,9 @@ env AP_PERIPH 1
 
 # enable watchdog
 
-# crystal frequency
-OSCILLATOR_HZ 12000000
+# crystal frequency set to 0 to use internal clock
+# OSCILLATOR_HZ 12000000
```
Comment (amilcarlucas) [inline libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef.dat:22]:
```suggestion
```

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef-bl.dat [hunk 4]
```diff
@@ -15,8 +15,9 @@ APJ_BOARD_ID 1053
 # setup build for a peripheral firmware
 env AP_PERIPH 1
 
-# crystal frequency
-OSCILLATOR_HZ 12000000
+# crystal frequency set to 0 to use internal clock
+# OSCILLATOR_HZ 12000000
```
Comment (amilcarlucas) [inline libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef-bl.dat:19]:
```suggestion
```

**Review by amilcarlucas — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef-bl.dat (hunk 1 above)
Comment (Lokesh-Carbonix) [reply libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef-bl.dat:64]: Thanks to that I will make the changes.
Should I make the changes and create a new pull request (to keep fewer commits)?
or should I add the changes with new commits here itself?
What is preferred?

Comment (hendjoshsr71) [reply libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef-bl.dat:64]: @Lokesh-Carbonix  Makes the changes as an additional commit and then squash the commits to one commit.

Just a note. We consider a manufacturers HWDef to be theirs. AKA we just trust you to get it correct. Especially for boards in progress like I think this one is?

Changes to boards out in general use get more scrutiny.

Comment (Lokesh-Carbonix) [reply libraries/AP_HAL_ChibiOS/hwdef/CarbonixL496/hwdef-bl.dat:64]: Thanks, @hendjoshsr71 I will squash it. Yes, I understand on HWdef is the manufacturer's responsibility to test and this board is at an almost stable state.

**Comment by Lokesh-Carbonix** [comment]
@hendjoshsr71 have a look I have squashed the commits. I had to -f push though. Let me know if everything is correct.

**Comment by hendjoshsr71** [comment]
> @hendjoshsr71 have a look I have squashed the commits. I had to -f push though. Let me know if everything is correct.

Force push is the normal workflow for us when updating PR branches with fixes.

**Review by tridge — APPROVED** [review]

**Pull request merged**